pkg/server/objects_operations.go

```diff
@@ -2,38 +2,78 @@ package server
 
 import (
 	"context"
+	"time"
 
 	"github.com/codenotary/immudb/pkg/api/schema"
 	"github.com/codenotary/immudb/pkg/api/schemav2"
 	"github.com/golang/protobuf/ptypes/empty"
 )
 
 func (s *ImmuServer) LoginV2(ctx context.Context, loginReq *schemav2.LoginRequest) (*schemav2.LoginResponseV2, error) {
```

Comment:
what about OpenSession rather than login?

Reply:
Could be, but again (like in the next comment) its again a path to learn user about new concepts which he doesnt know in REST

Reply:
Resolved in 
https://github.com/codenotary/immudb/pull/1546